assets/js/templates/sidebar-items.handlebars

```diff
@@ -34,6 +34,20 @@
             </li>
           {{/each}}
         {{else}}
+          {{#showSections node}}
+            <li class="docs open">
```

Comment:
Note that I rendered this open by default. I'm not sure if that's desired though.

Reply:
I think this would be ok only if this is the current page. Otherwise, it will probably make navigation difficult. WDYT?

Reply:
that makes sense to me, i'll make that change.